src/Facebook/HttpClients/FacebookPsrHttpClient.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Facebook\HttpClients;
+
+use Facebook\Http\GraphRawResponse;
+use Nyholm\Psr7\Factory\Psr17Factory;
+use Psr\Http\Client\ClientInterface;
+use Psr\Http\Message\RequestFactoryInterface;
+use Psr\Http\Message\StreamFactoryInterface;
+use Symfony\Component\HttpClient\Psr18Client;
+
+class FacebookPsrHttpClient implements FacebookHttpClientInterface
```

Comment:
let's replace this by `FacebookSymfonyHttpClient`
PSR18 doesn't handle timeouts, so it cannot be correctly used here (at least we don't have to push it)